docs/platform_concepts/growth/dataexplorer/customtables.md

```diff
@@ -5,7 +5,7 @@ sidebar_label : Analyse custom tables
 
 #### What are custom tables(datasets)?
 
-The bot builder can create Custom datasets that are tailored to your needs. You can modify these tables according to your preferences using a **custom query**.
+On **Studio** you can create [Custom datasets](https://docs.yellow.ai/docs/platform_concepts/studio/database) that are tailored to your needs (ex- these tables have the data obtained from variables from the flows). You can modify these tables according to your preferences using a **Custom query**.
```

Comment:
Hey, it's highly recommended to use the full forms of phrases for clarity. Instead of using "ex", it is better to use "for example" or the widely recognized  "e.g.,". Using "ex" can be misleading